### ISSUE TITLE
Can't find variable: exports

### ISSUE THREAD

**Issue by moshmage** [monounity/karma-typescript#120]
So. This beast was testing, at least 4 months ago. Now, not so much. I haven't changed anything, I'm thinking `karma-typescript` and `typescript` itself *did* change quite a bit.

[whole repo](http://gitlab/moshmage/rxjs-socket.io)

tsconfig.json
```json
{
  "compilerOptions": {
    "module": "commonjs",
    "target": "es5",
    "sourceMap": false,
    "removeComments": false,
    "declaration": true,
    "moduleResolution": "node",
    "outDir": "dist/"
  },
  "include": [
    "src/**/*"
  ],
  "exclude": [
    "**/*.spec.ts"
  ]
}
```
karma-conf.js
```js
module.exports = function(config) {
    config.set({

        frameworks: ["jasmine", "karma-typescript"],

        files: [
            { pattern: "src/**/*.ts" }
        ],

        preprocessors: {
            "**/*.ts": ["karma-typescript"]
        },

        reporters: ["progress", "karma-typescript"],


        karmaTypescriptConfig: {
            tsconfig: 'tsconfig.spec.json',
            reports: {
                "html": "coverage",
                "text-summary": "" // destination "" will redirect output to the console
            }
        },
        browsers: ["PhantomJS"]
    });
};
```

What once was passing with that configuration, now it's returning 

```logtalk
PhantomJS 2.1.1 (Windows 8 0.0.0) ERROR
  ReferenceError: Can't find variable: exports
  at src/index.js:5:0 <- src/index.js:24
```

at every `it`

**Comment by erikbarke** [comment]
Hey @moshmage, there were breaking changes in Typescript that were fixed in `karma-typescript@3.0.0`, I got your repo working by installing the latest:

```bash
npm install karma-typescript@3.0.1
```

And I also had to exclude two packages in the Karma config:

```js
module.exports = function(config) {
    config.set({

        frameworks: ["jasmine", "karma-typescript"],

        files: [
            { pattern: "src/**/*.ts" }
        ],

        preprocessors: {
            "**/*.ts": ["karma-typescript"]
        },

        reporters: ["progress", "karma-typescript"],

        karmaTypescriptConfig: {
            bundlerOptions: {
                exclude: ["bufferutil", "utf-8-validate"]
            },
            tsconfig: 'tsconfig.spec.json',
            reports: {
                "html": "coverage",
                "text-summary": "" // destination "" will redirect output to the console
            }
        },
        browsers: ["PhantomJS"]
    });
};
```
The tests run now, but I get a couple of Typescript errors, both when compiling with `./node_modules/.bin/tsc --project tsconfig.spec.json` and when running `karma-typescript`, Typescript barfs on properties that aren't accessible etc.

**Comment by moshmage** [comment]
Oh! Makes sense. I did try to update Typescript but completly forgot about *karma*-stypescript \^\^'

>  Typescript barfs on properties that aren't accessible etc.
Yeah, that's bad design on my part but fully expected :)
